test/Microsoft.ML.Tests/Transformers/NAIndicatorTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Data.IO;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Tools;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Tests.Transformers
+{
+    public class NAIndicatorTests : TestDataPipeBase
+    {
+        private class TestClass
+        {
+            public float A;
+            public double B;
+            [VectorType(2)]
+            public float[] C;
+            [VectorType(2)]
+            public double[] D;
+        }
+
+        public NAIndicatorTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void NAIndicatorWorkout()
+        {
+            var data = new[] {
+                new TestClass() { A = 1, B = 3, C = new float[2]{ 1, 2 } , D = new double[2]{ 3,4} },
+                new TestClass() { A = float.NaN, B = double.NaN, C = new float[2]{ float.NaN, float.NaN } , D = new double[2]{ double.NaN,double.NaN}},
+                new TestClass() { A = float.NegativeInfinity, B = double.NegativeInfinity, C = new float[2]{ float.NegativeInfinity, float.NegativeInfinity } , D = new double[2]{ double.NegativeInfinity, double.NegativeInfinity}},
+                new TestClass() { A = float.PositiveInfinity, B = double.PositiveInfinity, C = new float[2]{ float.PositiveInfinity, float.PositiveInfinity, } , D = new double[2]{  double.PositiveInfinity, double.PositiveInfinity}},
+                new TestClass() { A = 2, B = 1, C = new float[2]{ 3, 4 } , D = new double[2]{ 5,6}},
+            };
+
+            var dataView = ComponentCreation.CreateDataView(Env, data);
+            var pipe = new NAIndicatorEstimator(Env,
+                new NAIndicatorTransform.ColumnInfo("A", "NAA"),
+                new NAIndicatorTransform.ColumnInfo("B", "NAB"),
+                new NAIndicatorTransform.ColumnInfo("C", "NAC"),
+                new NAIndicatorTransform.ColumnInfo("D", "NAD"));
+            TestEstimatorCore(pipe, dataView);
+            Done();
+        }
+
+        [Fact]
+        public void NAIndicatorStatic()
```

Comment:
>NAIndicatorStatic [](start = 20, length = 17)

Please move next to other static tests

Reply:
We decided to group static tests together and other transformer tests together

---
In reply to: [224861996](https://github.com/dotnet/machinelearning/pull/1217#discussion_r224861996) [](ancestors = 224861996)